core/txpool/legacypool/legacypool.go

```diff
@@ -2038,3 +2082,50 @@ func (t *lookup) RemotesBelowTip(threshold *big.Int) types.Transactions {
 func numSlots(tx *types.Transaction) int {
 	return int((tx.Size() + txSlotSize - 1) / txSlotSize)
 }
+
+// transferTransactions mainly moves from OverflowPool to MainPool
+func (pool *LegacyPool) transferTransactions() {
+	maxMainPoolSize := int(pool.config.GlobalSlots + pool.config.GlobalQueue)
+	extraSizeMainPool := maxMainPoolSize - int(uint64(len(pool.pending))+uint64(len(pool.queue)))
+	if extraSizeMainPool <= 0 {
+		return
+	}
+
+	currentMainPoolSize := pool.all.Slots()
+	canTransferOverflowPoolToMainPool := maxMainPoolSize > currentMainPoolSize
```

Comment:
`canTransferOverflowPoolToMainPool` better to simplify these variable names, make it shorter